Article/Article.js

```diff
@@ -1,20 +1,25 @@
 // Because classes are not hoised you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
 class Article {
-  constructor(domElement) {
+  constructor(article) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.article = article;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = article.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = 'Expand';
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', () => this.expandArticle());
   }
 
   expandArticle() {
     // Using our reference to the domElement, toggle a class to expand or hide the article.
-
+    this.article.classList.toggle('article-open');
+    if (this.expandButton.textContent === 'Expand') {
+      this.expandButton.textContent = 'Close';
+    } else {
+      this.expandButton.textContent = 'Expand';
+    };
```

Comment:
```suggestion
     expandArticle() {
    this.domElement.classList.toggle('article-open');
  }
```

// unless you're doing something custom here; this is all you need for functionality.